Ghidra/Features/Base/src/main/java/ghidra/app/util/viewer/field/AnnotatedStringHandler.java

```diff
@@ -95,4 +95,11 @@ public boolean handleMouseClick(String[] annotationParts, Navigatable sourceNavi
 	 * @return the example of how this is used.
 	 */
 	public String getPrototypeString();
+	
+	/**
+	 * Returns an example string of how the annotation is used
+	 * @param contained The text that may be wrapped
```

Comment:
'contained' should be renamed to 'displayText' to match the verbiage used elsewhere, such as in the help.   (Change this in each subclass too.)